src/lib/get-packer-repo-ref-for-plugins.ts

```diff
@@ -0,0 +1,18 @@
+import { cachedGetProductData } from 'lib/get-product-data'
+import { RootDocsPath } from 'types/products'
+import { getBetaLatestVersionRef } from 'views/docs-view/server'
+
+/**
+ * Resolves the `hashicorp/packer` branch
+ * from which `plugins-manifest.json`, `plugins-nav-data.json`,
+ * and Packer `/plugins` root path content will be loaded.
+ */
+export function getPackerRepoRefForPlugins() {
```

Comment:
Set this up to try to make the logic very clear for the ref we use for `dev-dot` Packer plugins.